sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -308,6 +308,99 @@ export function getCountDetailsOrUndefined(value: any): MessageCountDetails | un
   };
 }
 
+/**
+ * Service expects the time duration in ISO-8601 time duration format.
+ * Example: PT10M, P106DT2H48M5.47S
+ *
+ * This helper method is to convert the duration in seconds to ISO-8601 duration format.
+ *
```

Comment:
Can we mention here that we don't support weeks/months/years because the service doesn't send that?

Reply:
Good point. I've added it in the comments for one of the methods, I'll add it in the docs too.